gamestonk_terminal/stocks/fundamental_analysis/dcf_model.py

```diff
@@ -224,234 +224,187 @@ def others_in_sector(ticker: str, sector: str, industry: str) -> List[str]:
     industry = industry.replace("—", " - ")
     industry = industry.replace("/", " ")
 
-    sister_ticks = list(
-        fd.select_equities(country="United States", sector=sector, industry=industry)
+    sister_ticks = fd.select_equities(
+        country="United States", sector=sector, industry=industry
```

Comment:
You are looking at only companies in the United States here. Why exactly? If I would look at for example Taiwan Semiconductors, I would want to compare with companies like ASML and Micron. Both located in a different country. I think you should exclude the country parameter.

Reply:
This was due to a misunderstanding on my end. Removed!